pkg/controller/syncmachineset/gcp/gcpactuator.go

```diff
@@ -0,0 +1,146 @@
+package gcp
+
+import (
```

Comment:
Thanks for the migration, with the file in the folder of `gcp`, I think we can rename the file to `actuator.go`?